src/service.rs

```diff
@@ -62,13 +49,17 @@ pub struct DaServiceConfig {
     pub node_password: String,
 
     // network of the bitcoin node
-    pub network: Option<String>,
+    pub network: String,
 
     // taproot address that holds the funds of the sequencer
-    pub address: Option<String>,
+    // will be used as the change address for the inscribe transaction
+    pub address: String,
 
     // da private key of the sequencer
     pub sequencer_da_private_key: Option<String>,
+
+    // number of last paid fee rates to average if estimation fails
+    pub fee_rates_to_avg: usize,
```

Comment:
Let's make this Option and have an enum that makes Network -> DefaultFeeRate fallback

Reply:
this is the number of last paid fee rates to use when you can't get your normal feerate

What do you mean when you say Network -> DeafualtFeeRate fallback?

Reply:
DAServiceConfig is given when the rollup is started running. User has to input a fee_rates_to_avg. If this is not intended you should remove it from the config and only keep it in the struct.

If so use a default value per network to initiate this var of service struct